draft-ietf-ppm-dap.md

```diff
@@ -816,7 +816,8 @@ enc, payload = SealBase(pk,
 where `pk` is the aggregator's public key; `server_role` is the Role of the
 intended recipient (`0x02` for the leader and `0x03` for the helper), `task_id`
 is the task ID, `metadata` is the report metadata, and `input_share` is the
-Aggregator's input share.
+Aggregator's input share. The `SealBase()` function is as specified in
+{{!HPKE}} for the ciphersuite indicated by the HPKE configuration.
```

Comment:
The strings "SealBase" and "OpenBase" never actually occur in RFC 9180. Instead, [section 6.1](https://www.rfc-editor.org/rfc/rfc9180.html#section-6.1) describes "Seal<MODE>" and "Open<MODE>" and then enumerates the possible modes, among which is "Base". Just to make this a little easier on folks reading DAP, could we refer specifically to RFC 9180 section 6.1?

Reply:
Sure, mr. pedantic-pants

Reply:
(seriously, great idea.)